Exercise_7/scripts/GameFlow.js

```diff
@@ -0,0 +1,56 @@
+import KeyActions from './ticTacToe/KeyActions';
+import Board from './ticTacToe/Board';
+import EndOfGameCheck from './ticTacToe/EndOfGameCheck';
+import {SPACE_KEY, LEFT_KEY, RIGHT_KEY, UP_KEY, DOWN_KEY} from './ticTacToe/Constraints';
+
+(() => {
+    const DEFAULT_DELAY = 300;
+    let currentSign = 'X';
+    const fields = ['', '', '', '', '', '', '', '', ''];
+
+    const highlightField = (fieldIndex) => {
```

Comment:
But still, original way is better in some cases, even if its longer.

```javascript
const obj = {
  foo: 42,
  bar: function(){
    console.log(this.foo);
  }
}
obj.bar(); //42
const bar = obj.bar;
bar(); //undefined
```
So, if method of Board will use `this` context, it will be lost in your way, but Lukasz's way will work in any case. I think it will be better to stay all as it is, or replace lines
```javascript
[LEFT_KEY]: highlightField
```
to 
```javascript
[LEFT_KEY]: fieldIndex => Board.highlightField(fieldIndex)
```

Reply:
Ok. I agree.